assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -231,9 +236,23 @@ function APIProvider({ children }) {
     [link]
   );
 
-  const getAllUsers = useCallback(() => {
-    return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
-  }, [users]);
+  const getAuthors = useCallback(
```

Comment:
So you are now limiting the users to just getting 100 users? This doesn't make a lot of sense. Before it was getting all users by loading -1, which has a special bit of code to call the rest api recursively until it gets all users. Without that, you are limited to the first 100.

Reply:
The idea is to load 100 users by default for performance reasons, and then use the search to find more users.

Loading all users with -1 is exactly what we want to prevent / stop doing

Reply:
Yes, @spacedmonkey, as required, this now works in a similar way as the fontpicker -- the first 100 users a loaded by default and the rest of the users can be searched.